bls/bls.go

```diff
@@ -97,7 +98,7 @@ func ToBytes(E *BN254.ECP) []byte {
 
 func ToBytes2(E *BN254.ECP2) []byte {
 	const MFS = BN254.MFS
-	const G1S = 2*MFS + 1 /* Group 1 Size */
+	// const G1S = 2*MFS + 1 /* Group 1 Size */
```

Comment:
Please, remove all comments from your code. Except comments for function headers

Reply:
since this is golangci-lint problem, I'll fix in the next commit.